doc/tutorials/windows.md

```diff
@@ -32,4 +32,65 @@ window.WindowStartupLocation = WindowStartupLocation.CenterOwner;
 
 // Use Show or ShowDialog to show the window
 window.ShowDialog();
+```
+
+# [PowerShell](#tab/tabpowershell)
+```powershell
+$windowCreationOptions = New-Object Playnite.SDK.WindowCreationOptions
+$windowCreationOptions.ShowMinimizeButton = $false
+
+$window = $PlayniteApi.Dialogs.CreateWindow($windowCreationOptions);
+$window.Height = 768;
+$window.Width = 768;
+$window.Title = "Some title";
+
+# Set content of a window. Can be loaded from xaml, loaded from UserControl or created from code behind
+[xml]$xaml = @"
+<UserControl
+    xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+    xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml">
+
+    <UserControl.Resources>
+        <Style TargetType="TextBlock" BasedOn="{StaticResource BaseTextBlockStyle}" />
+    </UserControl.Resources>
+
+    <StackPanel Margin="15,0,0,0">
+        <StackPanel Orientation="Horizontal">
+            <TextBlock Text="An example button:" Margin="0,0,15,0" VerticalAlignment="Center"/>
+            <Button x:Name="MyButton" Content="Click me!"/>
+        </StackPanel>
+        <TextBlock Text="Currently selected games:" Margin="0,15,0,0"/>
+        <ItemsControl ItemsSource="{Binding}">
+            <ItemsControl.ItemTemplate>
+                <DataTemplate>
+                    <TextBlock Text="{Binding Path=Name}" Margin="15,0,0,0"
+                        Style="{StaticResource BaseTextBlockStyle}"/>
+                </DataTemplate>
+            </ItemsControl.ItemTemplate>
+        </ItemsControl>
+        <StackPanel Orientation="Horizontal" Margin="0,15,0,0">
+            <TextBlock Text="Total selected games: "/>
+            <TextBlock Text="{Binding .Count}"/>
+        </StackPanel>
+    </StackPanel>
+</UserControl>
+"@
+$reader = [System.Xml.XmlNodeReader]::new($xaml)
+$window.Content = [Windows.Markup.XamlReader]::Load($reader)

```

Comment:
I added a new PowerShell example using `PlayniteApi.Dialogs.CreateWindow()` to demonstrate this API. I used my old XAML test code adapted to use a `UserControl` as the example.

I like using an XAML example like this to demonstrate this feature, but I don't mind if we tweak the exact contents of this sample XAML. I think a complete example should probably be added to the C# tab too.